package.json

```diff
@@ -258,6 +259,7 @@
     "@types/semver": "^7.3.4",
     "@types/yargs": "^16.0.0",
     "bottlejs": "^2.0.0",
+    "ejs": "^3.1.6",
```

Comment:
I was thinking more `renderToString` style from `react-dom` (which already is a dependency), but I guess that would work too. I don't think `yarn.lock` was updated, how much does inclusion of `ejs` costs us in terms of extra dependencies?

Reply:
ejs was already included through another dependency. I will also check out react-dom renderToString you mentioned.

Reply:
@krassowski I looked into ReactDOMServer.renderToString. it doesn't look like an easy integration for our use case. we generate the HTML of the dialogs on the app backend using node.js. Dialog front-ends are quite basic HTML and JS. We would need to use react for dialog front-ends which would be significant overhead just for sanitize purposes. Also, sanitization would happen on the front-end which may not be as secure as intended.

Please let me know if I missed something and if you see an easier integration of ReactDOMServer.renderToString.